crates/forge/bin/cmd/compiler.rs

```diff
@@ -94,13 +94,17 @@ impl ResolveArgs {
                         })
                         .collect();
 
-                    (version.clone(), paths)
+                    (
+                        version.clone(),
+                        EvmVersion::default().normalize_version_solc(version).unwrap_or_default(),
```

Comment:
for consistency `forge compiler r -v --json` should not include the `EvmVersion`, only with `-vv`